index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles/index.css">
+  <title>Survey Form</title>
+</head>
+<body>
+  <h1 id="title">Survey Form</h1>
+  <p id="description">Thank you for taking the time to help us improve the platform</p>
+  <form id="survey-form" action="">
+    <label for="name" id="name-label">Name</label><br>
+    <input type="text" id="name" name="name" placeholder="Enter your name" required><br>
+    <label for="email" id="email-label">Email</label><br>
+    <input type="email" id="email" name="email" placeholder="Enter your email" required><br>
+    <label for="number" id="number-label">Phone Number </label><br>
+    <input type="number" id="number" name="number-label" min="10" max="99" placeholder="10" required><br>
+    <label for="opt">Which option best describes your current role?</label><br>
+    <select name="opt" id="dropdown">
+      <option value="">Select current role</option>
+      <option value="fjob">Full Time Job</option>
+      <option value="flearner">Full Time Learner</option>
+    </select><br>
+    <label for="email-label">Would you recommend freeCodeCamp to a friend?</label><br>
+    
+    <table>
+      <tr>
+        <td>    
+ <input type="radio" name="recom" value="recom" checked></td> 
+   <td>    
+<label for="recom">Definitely</label><br>
+ </td> 
+   </tr>
+      <tr>
+        <td> <input type="radio" name="recom" value="recom">
+        </td>
+        <td>    <label for="recom">May be</label><br> </td>
+      </tr>
+      
```

Comment:
Kindly try to not add blank lines, spaces, or indentations without a reason.

https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs. 

Refer to the link above regarding best practices for HTML and CSS